tools/generate-helpers/pg-table-bindings/list_test.go

```diff
@@ -19,3 +24,17 @@ func TestIsGloballyScoped(t *testing.T) {
 	assert.True(t, isGloballyScoped("*storage.Policy"))
 	assert.True(t, isGloballyScoped("storage.Policy"))
 }
+
+func TestIsDirectlyScoped(t *testing.T) {
+	for typ, directlyScoped := range map[proto.Message]bool{
+		&storage.NamespaceMetadata{}: true,
+		&storage.Cluster{}:           true,
+		&storage.Deployment{}:        true,
+		&storage.Image{}:             false,
+		&storage.CVE{}:               false,
```

Comment:
Could you add tests for a globally scoped resource type (e.g. Policy) ?
Could you also add a test for a fake/dummy resource type ?